astroplan/skycalc.py

```diff
@@ -0,0 +1,188 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+# Standard library
+import json
+import os
+
+# Third-party
+from astropy.io import fits
+import astropy.units as u
+
+# Local
+from .exceptions import SkyCalcError
+
+
+__all__ = ['skymodel']
+
+
+def skymodel(airmass=1.0, pwv_mode='pwv', season=0,
```

Comment:
To follow up: did the SkyCalc CLI folks ever get back to you via email about adding it to astroquery? I still think that's the best way to go for the future of this particular method.

Reply:
They did! They said I should feel free to go ahead and make a PR to astroquery, and that their "software developers would surely consider it" - not sure if that means they'd chime in on my PR or if they'd consider making their own PR at some point, but either way it sounds like they gave me the green light.